Tests/test_causalvelocityfilter.py

```diff
@@ -1,56 +1,35 @@
 import math
-import numpy as np
 import unittest
-import matplotlib.pyplot as plt
+
```

Comment:
Why so many extra carriage returns at the top?

Reply:
PEP8:

> Imports should be grouped in the following order:
> 
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.
> 
> You should put a blank line between each group of imports.